examples/requirements.txt

```diff
@@ -0,0 +1,2 @@
+# All requirements
+# python>=3.7
```

Comment:
maybe we should write for now 3.7<=python<=3.9 until we solve the compatibility issues? 
(also relevant for the readme)